extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -97,6 +103,22 @@ function jetpack_mailchimp_block_load_assets( $attr ) {
 						name="email"
 					/>
 				</p>
+				<?php foreach ( is_array( $values['interests'] ) ? $values['interests'] : array() as $interest ) : ?>
+					<input
+						name="interests[<?php echo esc_attr( $interest ); ?>]"
+						type="hidden"
+						class="mc-submit-param"
+						value=1
+					/>
+				<?php endforeach; ?>
+				<?php if ( $values['signupFieldTag'] && strlen( $values['signupFieldTag'] ) && $values['signupFieldValue'] && strlen( $values['signupFieldValue'] ) ) : ?>
```

Comment:
Could we maybe simplify this into something like this?

```suggestion
				<?php
				if (
					! empty( $values['signupFieldTag'] )
					&& ! empty( $values['signupFieldValue'] )
				) :
					?>
```

Reply:
Done: https://github.com/Automattic/jetpack/pull/14050/commits/d0b9121e7df285a9001eae389069b2d5e0c73107